modules/media/jsx/mediaIndex.js

```diff
@@ -1,119 +1,174 @@
-import React, {Component} from 'react';
 import Loader from 'Loader';
-import FilterForm from 'FilterForm';
+import FilterableDataTable from 'FilterableDataTable';
 import {Tabs, TabPane} from 'Tabs';
 
 import MediaUploadForm from './uploadForm';
-import formatColumn from './columnFormatter';
 
-class MediaIndex extends Component {
+class MediaIndex extends React.Component {
   constructor(props) {
     super(props);
-    loris.hiddenHeaders = ['Cand ID', 'Session ID', 'Hide File', 'File Type'];
 
     this.state = {
+      data: {},
       isLoaded: false,
-      filter: {},
     };
 
-    // Bind component instance to custom methods
+    /**
+     * Bind component instance to custom methods
+     */
     this.fetchData = this.fetchData.bind(this);
-    this.updateFilter = this.updateFilter.bind(this);
-    this.resetFilters = this.resetFilters.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+    this.formElements = this.formElements.bind(this);
   }
 
   componentDidMount() {
-    this.fetchData();
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}))
+      .catch((error) => console.error(error));
   }
 
   /**
-   * Retrieve data from the provided URL and save it in state
-   * Additionally add hiddenHeaders to global loris variable
+   * Retrive data from the provided URL and save it in state
+   * Additionaly add hiddenHeaders to global loris vairable
```

Comment:
Additionally was also right before

Reply:
That's what I get when I don't copy and paste